File: SuperHero/app/src/main/java/pe/isil/superhero/HeroService.java
package pe.isil.superhero;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface HeroService {

    @GET("10157703717092094/search/{heroName}")
    Call<HeroResponse> searchHero(@Path("heroName") String heroName);
}

File: SuperHeroCompose/app/src/main/java/pe/isil/superherocompose/data/remote/HeroService.kt
package pe.isil.superherocompose.data.remote

import pe.isil.superherocompose.data.model.HeroResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Path

interface HeroService {

    @GET("10157703717092094/search/{heroName}")
    fun searchHero(@Path("heroName") name: String): Call<HeroResponse>

}